app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -11,22 +11,22 @@
     </div>
   </div>
 
-<p>If you’re studying for a degree and interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
+<p>If you’re studying for an undergraduate degree or a master's degree and are interested in a career in teaching, an internship could help you to understand what it’s really like in the classroom. You’ll get to experience a range of activities to help you get a feel for school life.</p>
 
-<p>Internships last 3 weeks, start in June and you’ll be paid £300 per week.</p>
+<p>Internships last 3 weeks, start in June or July and you’ll be paid £300 per week.</p>
 
 <section class="col col-640">
   <%= render Content::QuoteComponent.new(
     text: "The support I had was amazing, I was treated like a member of staff, and I was welcomed by everyone.",
     name: "Chloe, former teaching intern",
-    background: "grey",
+    background: "yellow",
     large: false
   ) %>
 </section>
 
-<h2>Internship subjects</h2>
+<h2>Who can apply</h2>
 
-<p>Teaching internships are only available if you're studying for a related degree and interested in teaching certain subjects. In 2023, these included:</p>
+<p>You'll need to be studying for a related undergraduate or a master's degree and interested in teaching one of these subjects:</p>
 <ul>
   <li>chemistry</li>
   <li>computing</li>
```

Comment:
we've got one word links on 'maths' and 'physics' - ordinarily, we want to avoid one word links cos they can look a bit spammy, and I wonder if it looks a bit strange to link to pages on maths and physics but not the other subjects?